packages/api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -197,6 +195,31 @@ const isOrderFormStale = (form: OrderForm) => {
   return newEtag !== oldEtag
 }
 
+// Returns the regionalized orderForm
+const getOrderForm = async (
+  id: string,
+  session: Maybe<IStoreSession> | undefined,
```

Comment:
```suggestion
  session?: Maybe<IStoreSession>,
```

Reply:
ts breaks saying we can only use `?` in the endd of parameters